client/src/epics_db/atlas_to_local.py

```diff
@@ -0,0 +1,39 @@
+import json
+import math
+import os
+import requests
+
+BL_API_KEY = str(os.environ.get('BL_API_KEY'))
+
+f = open('epics_happi_db.json')

```

Comment:
Let's do `with open()` instead of open()?

Reply:
Done.